metrics/cgroups/v2/io.go

```diff
@@ -0,0 +1,111 @@
+// +build linux
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package v2
+
+import (
+	"strconv"
+	"strings"
+
+	v2 "github.com/containerd/containerd/metrics/types/v2"
+	metrics "github.com/docker/go-metrics"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var ioMetrics = []*metric{
+	{
+		name:   "io_rbytes",
+		help:   "IO bytes read",
+		unit:   metrics.Bytes,
+		vt:     prometheus.GaugeValue,
+		labels: []string{"major:minor"},
```

Comment:
why `:`?

Reply:
To make it closer to original, in the io.state the format is:
```
Maj:Min rbytes wbytes rios wios
```

Reply:
I think we should split

Reply:
easy